### DISCUSSION TITLE
Implementing symbols which get used in programming synthax

### DISCUSSION THREAD

**Discussion by kouyateb** [aradzie/keybr.com#89] [Ideas]
I was thinking of a feature for programmers. Where the text would include spcial symbols like { } , [ ] and so on. It masy even looked like real synthax to make it more reality like. It would increase the tipping speed while coding.

**Comment by zakissimo** [comment]
I'm a programer and I would love to see that option!

Right with `Add punctuation characters`. Something like `Add symbols` `{}[]()\\|+=-_*&^%$#@\><``.
Or maybe split it in categories, brackets, operators and the rest ? I don't know.

Thanks for this amazing work btw

**Comment by mercmobily** [comment]
I think this would be a fantastic idea.
